src/intel_driver.h

```diff
@@ -183,22 +195,33 @@ struct intel_driver_data {
     int locked;
 
     dri_bufmgr *bufmgr;
+    unsigned int gem_ctx_id;
+    drm_intel_context *gem_context;
 
     unsigned int has_exec2  : 1; /* Flag: has execbuffer2? */
     unsigned int has_bsd    : 1; /* Flag: has bitstream decoder for H.264? */
     unsigned int has_blt    : 1; /* Flag: has BLT unit? */
     unsigned int has_vebox  : 1; /* Flag: has VEBOX unit */
     unsigned int has_bsd2   : 1; /* Flag: has the second BSD video ring unit */
     unsigned int has_huc    : 1; /* Flag: has a fully loaded HuC firmware? */
+    unsigned int has_watchdog : 1; /* Flag: whether kmd has GPU watchdog support? */
 
     int eu_total;
 
     const struct intel_device_info *device_info;
     unsigned int mocs_state;
 };
 
+/* FIXME: this struct is currently private and part of libdrm */
+struct _drm_intel_context {
+    unsigned int ctx_id;
+    struct _drm_intel_bufmgr *bufmgr;
+};
+
```

Comment:
Is this structure used in your patch?

Reply:
You're right, I just removed struct _drm_intel_bufmgr and it still compiles just fine. I'll remove it and retest again on my testbed before uploading a new patch version.